observability/otel/otel_rn/otel-rn-3-2.adoc

```diff
@@ -0,0 +1,87 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="otel-rn-3-2"]
+= Release notes for {OTELName} 3.2
+:context: otel-rn-3-2
+
+toc::[]
+
+include::modules/otel-product-overview.adoc[leveloffset=+1]
+
+The {OTELName} is provided through the {OTELOperator}.
+
+////
+[id="otel-rn_3-2_cves"]
+== CVEs
+
+This release fixes link:https://access.redhat.com/security/cve/cve-202?-?????[CVE-202?-?????].
+////
+
+[id="otel-rn_3-2_technology-preview-features"]
+== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+// https://issues.redhat.com/browse/TRACING-3802 > https://docs.openshift.com/container-platform/4.14/observability/otel/otel_rn/otel-rn-past-releases.html#otel-rn_3-1_technology-preview-features
+////
+* The Target Allocator is an optional component of the OpenTelemetry Operator that shards Prometheus Receiver's scrape targets across the deployed fleet of OpenTelemetry Collector instances. The Target Allocator provides integration with the Prometheus `PodMonitor` and `ServiceMonitor` custom resources.
```

Comment:
@pavolloffay, could you please check if we even need to include the Target Allocator in the 3.2 release notes. Please see line 25 for the link to the published release notes for 3.1 and the fix version in the linked Jira.

Reply:
No, it is not listed here https://github.com/openshift/openshift-docs/pull/75558/files